tools/bridge/bridge/transfer_recorder.py

```diff
@@ -33,6 +33,23 @@ def __init__(self, minimum_balance: int) -> None:
         self.is_validator: Optional[bool] = None
         self.balance: Optional[int] = None
 
+    def log_current_state(self):
+        if self.is_validator:
+            validator_status = "validating"
+        else:
+            validator_status = "not validating"
+
+        balance_in_eth = (self.balance or 0) / 10 ** 18
```

Comment:
If I remember correctly, balance is `None` if it hasn't been checked yet. Printing `0` then could be confusing in that case. This probably doesn't happen too often, but it may if the node is offline.

Reply:
That's right. I didn't care enough to fix this. If the node is offline, the program will not start at the moment. So, this shouldn't be a problem in practice.